spec/services/alerts/framework_adapter_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+GAS_FUEL_ALERT_TYPE = AlertType.create(fuel_type: :gas, frequency: :weekly, class_name: 'Alerts::DummyAlertClass', description: 'Woof', title: 'Broken')
```

Comment:
Can we scope this constant better? It would be available to all tests where it is now.